docs/core/tools/project-json-to-csproj.md

```diff
@@ -0,0 +1,592 @@
+</PropertyGroup>
+```
+
+### Other common build options
+
+```json
+{
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "nowarn": ["CS0168", "CS0219"],
+    "xmlDoc": true,
+    "preserveCompilationContext": true,
+    "outputName": "Different.AssemblyName",
+    "debugType": "portable",
+    "allowUnsafe": true,
+    "define": ["TEST", "OTHERCONDITION"]
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  <NoWarn>$(NoWarn);CS0168;CS0219</NoWarn>
+  <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  <PreserveCompilationContext>true</PreserveCompilationContext>
+  <AssemblyName>Different.AssemblyName</AssemblyName>
+  <DebugType>portable</DebugType>
+  <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  <DefineConstants>$(DefineConstants);TEST;OTHERCONDITION</DefineConstants>
+</PropertyGroup>
+```
+
+## packOptions
+
+See also [Files](#files).
+
+### Common pack options
+```json
+{
+  "packOptions": {
+    "summary": "A bundle of cats",
+    "tags": ["hyperscale", "cats"],
+    "owners": [ "Nate", "Jenna" ],
+    "releaseNotes": "Version 1.0",
+    "iconUrl": "https://icons.com/awesomeness.png",
+    "projectUrl": "https://github.com/natemcmaster",
+    "licenseUrl": "https://www.apache.org/licenses/LICENSE-2.0",
+    "requireLicenseAcceptance": false,
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/natemcmaster/natemcmaster.github.io"
+    },
+    "owners": ["Fabrikam", "Microsoft"]
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <Summary>A bundle of cats</Summary>
+  <PackageTags>hyperscale;cats</PackageTags>
+  <PackageReleaseNotes>Version 1.0</PackageReleaseNotes>
+  <PackageIconUrl>https://icons.com/awesomeness.png</PackageIconUrl>
+  <PackageProjectUrl>https://github.com/natemcmaster</PackageProjectUrl>
+  <PackageLicenseUrl>https://www.apache.org/licenses/LICENSE-2.0</PackageLicenseUrl>
+  <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+  <RepositoryType>git</RepositoryType>
+  <RepositoryUrl>https://github.com/natemcmaster/natemcmaster.github.io</RepositoryUrl>
+  <!-- owners is not supported in MSBuild -->
+</PropertyGroup>
+```
+
+There is no equivalent for the `owners` element in MSBuild.
+
+## scripts
+
+```json
+{
+  "scripts": {
+    "precompile": "generateCode.cmd",
+    "postpublish": [ "obfuscate.cmd", "removeTempFiles.cmd" ]
+  }
+}
+```
+
+Their equivalent in MSBuild are targets:
+
+```xml
+<Target Name="MyPreCompileTarget" BeforeTargets="Build">
+  <Exec Command="generateCode.cmd" />
+</Target>
+
+<Target Name="MyPostCompileTarget" AfterTargets="Publish">
+  <Exec Command="obfuscate.cmd" />
+  <Exec Command="removeTempFiles.cmd" />
+</Target>
+```
+
+
+## runtimeOptions
+
+```json
+{
+  "runtimeOptions": {
+    "configProperties": {
+      "System.GC.Server": true
+    }
+  }
+}
+```
+
+All settings in this group should be placed in the project folder into a file called
```

Comment:
This section is outdated now. Here's the latest on configuring runtime options:

----------------------


```xml
<PropertyGroup>
  <ServerGarbageCollection>true</ServerGarbageCollection>
  <ConcurrentGarbageCollection>true</ConcurrentGarbageCollection>
  <RetainVMGarbageCollection>true</RetainVMGarbageCollection>
  <!-- I'm not suggesting these settings...just showing usage ;) -->
  <ThreadPoolMinThreads>10</ThreadPoolMinThreads>
  <ThreadPoolMaxThreads>100</ThreadPoolMaxThreads>
</ProeprtyGroup>
```

**Tip:** If you are working on a web project, specifying `<Project Sdk="Microsoft.NET.Sdk.Web">`
will automatically default ServerGarbageCollection to true. 

See [CoreCLR configuration](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/clr-configuration-knobs.md)
for information on these settings.